src/source/PeerConnection/PeerConnection.c

```diff
@@ -474,7 +474,66 @@ STATUS generateJSONSafeString(PCHAR pDst, UINT32 len)
     return retStatus;
 }
 
-STATUS createPeerConnection(PRtcConfiguration pConfiguration, PRtcPeerConnection *ppPeerConnection)
+STATUS rtcpReportsCallback(UINT32 timerId, UINT64 currentTime, UINT64 customData)
```

Comment:
Maybe I am missing something. I see this is being set up to run periodically in PeerConnection.c. We are enforcing report generation in this case and ignoring the case where the user might not want to see Rtp related reports in the logs. Is my understanding right? 

What I am eluding to is to give the user the choice to pull Rtp related stats periodically.

Reply:
this implements https://tools.ietf.org/html/rfc3550#section-6.4.1
rtcp sender reports are required according to rtp spec
metrics are calculated from these reports as per webrtc spec https://www.w3.org/TR/webrtc-stats/#dom-rtcremoteinboundrtpstreamstats

>  roundTripTime of type double
>    Estimated round trip time for this SSRC based on the RTCP timestamps in the RTCP Receiver Report (RR) and measured in seconds. Calculated as defined in section 6.4.1. of [RFC3550]. If no RTCP Receiver Report is received with a DLSR value other than 0, the round trip time is left undefined.

you cant have outbound and remote inbound metrics without sending rtcp sender reports

Reply:
Ah! Ok. My bad. I got confused.